app/js/views/app_view.js

```diff
@@ -1,15 +1,42 @@
 define([
   'jquery',
   'backbone',
-  'text!templates/app_view.ejs'
-], function ($, Backbone, indexTemplate) {
+  'text!templates/app_view.ejs',
+  'collections/todo_item_collection',
+  'models/todo_item'
+], function ($, Backbone, indexTemplate, TodoItemCollection, TodoItem) {
   return Backbone.View.extend({
     el: this.$('#todo-app'),
 
+    initialize: function(){
+      this.todoItemCollection = new TodoItemCollection();
+    },
+
+    events:{
+      "click #add-todo-button" : "addTodo",
+      "keydown #todo-title-textbox" : "onKeyDownEventHandler"
+    },
+
+    onKeyDownEventHandler: function(e){
+      if(e.keyCode==13){
+        this.$('#add-todo-button').click();
```

Comment:
This looks like a hack. 
1) what is the magic number 13 ? (we can pull it in a constant - ENTER_KEY_CODE)
2) don't trigger the click on the button. you end up triggering a lot of events and the code will be hard to maintain. use a common method for executing logic that is called from both event handlers (key press / click)